lib/matplotlib/axes/_axes.py

```diff
@@ -7772,13 +7772,10 @@ def hist2d(self, x, y, bins=10, range=None, density=False, weights=None,
 
         Notes
         -----
-        - Currently ``hist2d`` calculates its own axis limits, and any limits
```

Comment:
I think we should leave a `.. versionchanged::` notice here.

Reply:
done